maestro-cli/src/main/java/maestro/cli/runner/MaestroCommandRunner.kt

```diff
@@ -176,7 +190,16 @@ object MaestroCommandRunner {
             cachedAppState
         }
 
+        if (onFlowStart != null) {
+            orchestra.runFlow(onFlowStart.commands)
```

Comment:
In case if someone is using both init flow and onFlowStart what happens?

Reply:
in this case both will be executed, but I doubt someone will try to do that, plus the plan is to deprecate initFlow asap